lib/region/europe/index.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * @module Module for determining if a locale is in the EU.
+ *
+ * Examples:
+ *
+ * // Example 1, loading this module directly:
+ * var europe = require('bv-ui-core/lib/region/europe');
+ * europe.has('en_US'); // false
+ * europe.has('en_GB'); // true
+ *
+ * // Example 2, loading the region module:
+ * var region = require('bv-ui-core/lib/region');
+ * region.has('en_US', 'europe'); // false
+ * region.has('en_GB', 'europe'); // true
+ */
+
+var countryCodes = require('./countryCodes');
+
+module.exports = {
+
+  /**
+   * Determine if a given locale is an EU locale based on the country code.
+   *
+   * @param {String} locale - The locale to be tested. This function accepts
+   *   locales of both xx_XX format and country codes on their own, XX.
+   *
+   * @return {Boolean} Returns true if the locale is an EU locale.
+   *   Returns false otherwise.
+   */
+  has: function (locale) {
+
+    // First, we need to fish out a country code. If the locale is of the xx_XX
+    // format, we need to toss the language part and keep the country code. If
+    // the locale is just a country code already, we can just keep it. Either
+    // way, we can just create split along '_' and grab the last one from the
+    // resulting list.
+    var countryCode = locale.split(/[_\-]/).pop();
```

Comment:
FWIW, I think if someone does `europe.has('fr')`, this returns `false`. This is technically correct, but the string `fr` parses into a countryCode of `fr`, which may have misleading side-effects if someone is not careful. It might be worth considering erroring out if the locale doesn't match the pattern.